state/errors.go

```diff
@@ -171,3 +171,7 @@ func (e ErrCannotLoadState) Error() string {
 func (e ErrCannotLoadState) Unwrap() error {
 	return e.Err
 }
+
+var ErrFinalizeBlockResponseUnmarshalError = errors.New("problem retrieving the finalize block response")
+
+var ErrFinalizeBlockResponseCorruptedError = errors.New("cannot retrieve finalize block response, corrupted data or spec change")
```

Comment:
Can we add these in line 164-165?

Reply:
I will wrap this so these errors will change and will have Error() and Wrap() so will keep them separate